integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -120,14 +121,59 @@ context(`Change links on the Check Your Answers page when updating an Induction`
       .hasNoEducationalQualificationsDisplayed()
   })
 
-  // TODO - RR-736 Implement this test
-  it.skip('should support all Change links on a Long Question Set Induction', () => {
+  /* TODO - RR-736 finish the implementation of this test by clicking all Change links:
+   * Hoping To Work On Release (perhaps not do this one as it will turn the induction back into a short question set!)
+   * Educational Qualifications - Done
+   * Highest Level of Education - Done
+   * Other Training
+   * Worked Before
+   * Work history
+   * Type of work interested in
+   * Particular job roles
+   * Personal Skills
+   * Personal Interests
+   * Factors affecting ability to work
+   */
+  it('should support all Change links on a Long Question Set Induction', () => {
     // Given
     cy.updateLongQuestionSetInductionToArriveOnCheckYourAnswers(prisonNumber)
     const checkYourAnswersPage = Page.verifyOnPage(CheckYourAnswersPage) // eslint-disable-line @typescript-eslint/no-unused-vars
 
     // When
+    // Change Highest Level of Education
```

Comment:
And I've implemented the 2 education questions in the Long Question Set cypress test, which should be a good starter for 10